src/directory/directory.mjs

```diff
@@ -332,6 +332,9 @@ export const directory = {
                 {
                   path: 'src/pages/[platform]/build-a-backend/storage/use-aws-sdk/index.mdx'
                 },
+                {
+                  path: 'src/pages/[platform]/build-a-backend/storage/use-with-custom-s3/index.mdx'
```

Comment:
existing?

Reply:
I wanted to avoid conveying that you can import an existing bucket. Want to be more explicit about how they can use the libraries with non-Amplify managed resources